client/web/compose/src/views/Admin/Modules/Edit.vue

```diff
@@ -703,6 +703,16 @@ export default {
 
     handleFieldEdit (field) {
       this.updateField = compose.ModuleFieldMaker({ ...field })
+
+      if (this.updateField.kind === 'Select') {
```

Comment:
This is a very "bandage" kind of solution. 
I suggest you take a look at `lib/js/select.ts`, and see if you can persist the new property there so that compose.ModuleFieldMaker won't "remove" it

Only select.ts requires changes